base/src/main/java/io/spine/validate/FieldValidator.java

```diff
@@ -150,7 +151,9 @@ final IfInvalidOption ifInvalid() {
      *
      * <p>Do not call this method directly. Use {@link #validate() validate()} instead.
      */
-    protected abstract void validateOwnRules();
+    @SuppressWarnings("NoopMethodInAbstractClass"/* Does nothing for all but one subclasses. */)
```

Comment:
This is not fool-proof.  If there can be only one case of calling this method, please have a fried class that does this. Or, check the caller type. We have `InvocationGuard`, but its usage should be seriously weighted out.